File: baekjoon/js/1676.js
const fs = require("fs");
const buf = fs.readFileSync(0, "utf-8").toString().trim();
const n = Number(buf);

let num = 1n;

for (let i = 1; i <= n; i++) {
  num *= BigInt(i);
}

let result = 0;

while (true) {
  if (num % 10n !== 0n) {
    break;
  }

  result++;
  num /= 10n;
}

console.log(result);

File: baekjoon/rust/1676.rs
use std::io::stdin;

fn main() {
    let mut buf = String::new();
    stdin().read_line(&mut buf).unwrap();

    let n: u16 = buf.trim().parse().unwrap();

    let mut result = 0;
    let mut i = 5;

    while n >= i {
        result += n / i;
        i *= 5;
    }

    println!("{}", result);
}
